scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -2015,6 +2098,24 @@ then
 
     rm -f "${DATA}/${IST_FILE}"
 
+    # Get keyring component status
+    get_mysqld_path
+    plugin_dir=$WSREP_SST_OPT_PLUGINDIR
+    get_keyring_manifest_and_config "${DATA}" ${plugin_dir}
+
+    if [[ -z plugin_dir && -n $keyring_manifest_file ]]; then
+        wsrep_log_error "******************* FATAL ERROR ********************** "
+        wsrep_log_error "FATAL: The server is configured to use keyring, but the plugin_dir is not set."
+        wsrep_log_error "Please set the plugin_dir in the cnf file and retry."
```

Comment:
Is it necessary to put it in my.cnf? We are passing it from server via cmd line parameter to sst script. So it looks like passing it via cmd line to mysqld should work.

Reply:
Thanks for catching this. This is no more a requirement now since we pass the plugin-dir as a parameter to SST script. I will remove it.

Reply:
Done